homeassistant/components/sonos/speaker.py

```diff
@@ -40,7 +40,9 @@
 class SonosSpeaker:
     """Representation of a Sonos speaker."""
 
-    def __init__(self, hass: HomeAssistant, soco: SoCo, speaker_info: dict[str, Any]):
+    def __init__(
+        self, hass: HomeAssistant, soco: SoCo, speaker_info: dict[str, Any]
+    ) -> None:
```

Comment:
Return type for `__init__` is not necessary if it has at least 1 typed argument. IMO, it's cleaner without it.

Reply:
We should add it everywhere.